micro/reify_test.go

```diff
@@ -28,44 +27,44 @@ scheme code:
 	)
 */
 func TestReify(t *testing.T) {
-	a1 := "(,x . (,u ,w ,y ,z ((ice) ,z)))"
-	a2 := "(,y . corn)"
-	a3 := "(,w . (,v ,u))"
-	s := "(" + a1 + " " + a2 + " " + a3 + ")"
-	e, err := sexpr.Parse(s)
-	if err != nil {
-		t.Fatal(err)
-	}
+	u := ast.NewVar("u", 0)
+	v := ast.NewVar("v", 1)
+	w := ast.NewVar("w", 2)
+	x := ast.NewVar("x", 3)
+	y := ast.NewVar("y", 4)
+	z := ast.NewVar("z", 5)
+	a1 := ast.NewList(x, u, w, y, z, ast.NewList(ast.NewList(ast.NewSymbol("ice")), z))
+	a2 := ast.NewList(y, ast.NewSymbol("corn"))
```

Comment:
I think this needs to be ast.NewSExpr or ast.NewPair something, but this was an improper list before.  I don't know, if you know about improper lists, if not, reply and I will explain a bit here.